waiter/integration/waiter/kubernetes_scheduler_integration_test.clj

```diff
@@ -333,9 +333,9 @@
   (testing-using-waiter-url
     (when (using-k8s? waiter-url)
       (let [current-user (retrieve-username)]
-        (testing "No service account with default namespace"
```

Comment:
@DaoWen do you remember why we enforced this check?

Reply:
That's unfortunate. I think we just assumed this would be opt-in (i.e., add the `namespace` explicitly if you want it). If we're switching to "run as user by default", then we'll want to change this to just check if the namespace and run-as-user match.